.github/workflows/melos-ci.yml

```diff
@@ -0,0 +1,15 @@
+name: Iconica Standard Melos CI Workflow
+# Workflow Caller version: 1.0.0
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  call-global-iconica-workflow:
+    uses: Iconica-Development/.github/.github/workflows/melos-ci.yml@master
+    secrets: inherit
+    permissions: write-all
+    with:
+      subfolder: '.' # add optional subfolder to run workflow in
+      flutter_version: 3.24.3
```

Comment:
I prefer to not lock the version at all and if we do also add fvm configuration.
Ideally the package just works with both versions and we only start locking versions when a new release breaks CI or the package doesn't work anymore.